app/domain/usecase/get_deliveries.go

```diff
@@ -0,0 +1,44 @@
+package usecase
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/techhub-jf/farmacia-back/app/domain/dto"
+	"github.com/techhub-jf/farmacia-back/app/gateway/api/handler/schema"
+)
+
+type GetDeliveriesInput struct {
+	Pagination dto.Pagination
+}
+
+type GetDeliveriesOutput struct {
+	schema.ListDeliveriesOutput
```

Comment:
```suggestion
	Deliveries []entity.Delivery
	TotalDeliveries int
```

Acho que é melhor não usar o schema aqui para desacoplar o handler e o usecase. E montar o que é necessário de response lá no handler, onde enxergo que seria o melhor lugar, utilizando o input da request lá direto também para gerar o meta.